<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpWindow</class>
 <widget class="QWidget" name="SignUpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>회원가입</string>
  </property>
  <property name="styleSheet">
   <string>
    QWidget {
        background-color: #f9f9f9;
        font-family: Arial, sans-serif;
    }
    QPushButton {
        background-color: #87ceeb;
        border: 1px solid #4682b4;
        border-radius: 8px;
        color: white;
        font-weight: bold;
        padding: 5px 10px;
    }
    QPushButton:hover {
        background-color: #4682b4;
    }
    QLineEdit {
        border: 1px solid #87ceeb;
        border-radius: 5px;
        padding: 5px;
    }
    QLabel {
        font-size: 12px;
        color: #666666;
    }
   </string>
  </property>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
     QLabel {
         font-size: 24px;
         font-weight: bold;
         color: #4682b4;
     }
    </string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>아이디 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conform">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>비밀번호 입력</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>닉네임 입력</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="genderGroupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>성별</string>
   </property>
   <widget class="QRadioButton" name="checkBoxM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>남자</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="checkBoxF">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>여자</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
